src/popUp.js

```diff
@@ -1,28 +1,10 @@
 import React, { Component } from 'react';
 
 class PopUp extends Component {
-  constructor(props){
-    super(props)
-    this.close = this.close.bind(this)
-  }
-  state = {
-    display: "block",
-    title: "Henry's day at the park",
-    description: "A write up on Henry the Thirds trip to Central Park"
-  }
```

Comment:
Looks like you aren't using any of this state anymore - but note that if you want to set dummy data or default values, you can use [defaultProps](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) to get that behaviour.

```
PopUp.defaultProps = {
  title: 'a default title',
  description: 'some more default text'
}
```